README.md

```diff
@@ -2,14 +2,15 @@
 
 <img src="website/static/images/tag-contributor-strategy-icon-color.png" alt="a line drawing of a busy bee" align=right width="150px">
 
-TAG Contributor Strategy is responsible for contributor experience,
-sustainability, governance, and openness guidance to help CNCF community groups
+Contributor Strategy is a [Technical Advisory Group](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md) (TAG) responsible for contributor experience, sustainability, governance, and openness guidance to help CNCF community groups
 and projects with their own contributor strategies for a healthy project.
 
 If you are looking for how-to resources produced by the TAG, please visit
 [contribute.cncf.io](https://contribute.cncf.io/) or have a look at the
 contributor, goverance, and related [templates](https://github.com/cncf/project-template).
 
+For people new to this space, you can learn more about [CNCF TAG's](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md) by following the link.
```

Comment:
In general, I have a preference for not saying "follow this link" since it's implied anyway, but this is just my personal pet peeve, so I reworded it a bit to say more about what they could find out about the TAGs :)

Reply:
Agree. Updated. Thank you.

Reply:
Agree. Updated. Thank you.